app/uk/gov/hmrc/exports/migrations/repositories/MongoRepository.scala

```diff
@@ -53,14 +53,19 @@ abstract class MongoRepository private[migrations] (val mongoDatabase: MongoData
     }
 
   // Index is considered unique when:
-  // 1. Every uniqueField value is equal 1
-  // 2. "ns" field contains fullCollectionName
-  // 3. "unique" field is true
+  // 1. Name is "_id_" (these are implicitly unique https://docs.mongodb.com/manual/indexes/index.html#default-id-index)
+  // 2. Every uniqueField value is equal 1
+  // 3. "ns" field contains fullCollectionName
+  // 4. "unique" field is true
   private def isIndexUnique(index: Document): Boolean = {
     val key = index.get("key").asInstanceOf[Document]
+
+    if (index.getString("name").equals("_id_")) return true
```

Comment:
Its not adding an index. This is my fix to the existing migration code that was breaking because this function was reporting the automatic mongo '_id' index as not being unique. That resulted in the migration code trying to drop the '_id' index and that isn't allowed (so the whole thing failed).

Reply:
> because this function was reporting the automatic mongo '_id' index as not being unique. 
mongo _id is always unique. How did we end up with _id duplicates?

Reply:
Its not a duplicate, it is the original _id index that this method was mistakenly saying is not unique.

Reply:
So this is just being used to manage the `Changelog` collection?
Sorry... this is really confusing.